data/sql/updates/pending_db_world/rev_1556033308686302400.sql

```diff
@@ -0,0 +1,232 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1556033308686302400');
+
+-- Creating a new SmartAI script for [Creature] ENTRY 17701 (name: Lord Xiz)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17701;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 17701);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17701, 0, 0, 0, 0, 0, 100, 0, 2000, 2000, 32000, 32000, 11, 13318, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Lord Xiz - In Combat - Cast \'13318\'');
```

Comment:
This creature doesn't have abilities, remove it. [Creature] ENTRY 17701 (name: Lord Xiz)